src/libraries/Pool.sol

```diff
@@ -309,7 +309,7 @@ library Pool {
 
         bool exactInput = params.amountSpecified < 0;
 
-        if (swapFee == LPFeeLibrary.MAX_LP_FEE && !exactInput) {
+        if (swapFee >= LPFeeLibrary.MAX_LP_FEE && !exactInput) {
             InvalidFeeForExactOut.selector.revertWith();
         }
```

Comment:
crazy man ask -- gas should be cheaper if we put `!exactInput` as the first check (short-circuiting), because 50% of the swaps are exactInput and we dont need to revert for >100% swap fees

Reply:
wait nvm, we want to short circuit on the fee being too large, since its false like ~100% of the time